start/sw/mobile-app/mobile-installation.md

```diff
@@ -8,4 +8,20 @@ description: Creating mobile applications using Google Android Studio.
 
 # Mobile application
 
-Today wide range of smart mobile devices make it possible to create clever applications. See [Google Android Studio](/start/sw/mobile-app/mobile-devassistant.html) for a step-by-step instructions to install Android Studio and create your own ```hello world``` mobile application.
+Today wide range of smart mobile devices make it possible to create clever applications. 
+
+# Get started
+
+## Install required libraries
+
+`sudo dnf install glibc.i686 glibc-devel.i686 libstdc++.i686 zlib-devel.i686 ncurses-devel.i686`
+
+## Install Android Studio
+
+1. Download [Android Studio](https://developer.android.com/studio/index.html).
+
+2. Unpack the .zip and install to an appropriate location.
+
+3. Navigate to directory where Android Studio was installed and run with `./bin/studio.sh`
```

Comment:
Please use block code (see comment above) and mention what is done by this script.